components/datepicker/metadata/mods.md

```diff
@@ -0,0 +1,38 @@
+| Modifiable Custom Properties                            |
+| ------------------------------------------------------- |
+| `--mod-combobox-quiet-fieldbutton-border-radius`        |
```

Comment:
It looks like this one may need to be renamed to `--mod-datepicker-quiet-fieldbutton-border-radius` throughout the CSS? `--mod-quiet-button-offset` also looks like it should be `--mod-datepicker-quiet-button-offset`.

Unrelated to your changes: it looks like part of the Migration guide needs to change "Combobox now uses" to "Date picker now uses", if you could update that too.